beancount/parser/booking_full_test.py

```diff
@@ -2424,17 +2594,14 @@ def test_ambiguous__AVERAGE__simple_merge2_match1(self, _, __):
           Assets:Account        -49 HOOL {}
 
         2015-06-01 * #booked #reduced
-          M Assets:Account      -50 HOOL {100.00 USD, 2015-10-01}
```

Comment:
Explain this to make sure we're on the same wavelength... because the original two lots already got merged on application of the first transaction?
(I think I know why I used to have those: in the original test development, I assumed that I'd only be merging on reductions.)